src/components/NavBar/Search/Search.component.jsx

```diff
@@ -1,18 +1,55 @@
 import SearchIcon from '@material-ui/icons/Search';
-import React from 'react';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import CancelIcon from '@material-ui/icons/Cancel';
+import { IconButton } from '@material-ui/core';
 import { StyledInput, IconContainer, SearchContainer } from './Search.styled';
 
-const Search = ({ value }) => {
+const Search = ({ handleSearchChange }) => {
+  const history = useHistory();
+  const [keyWord, setKeyWord] = useState('');
+
+  const handleChange = (event) => {
+    setKeyWord(event.target.value);
+  };
+
+  const triggerChange = () => {
+    if (!keyWord) history.push('/');
```

Comment:
why do we need to do `history.push('/');` here. I suggest that if `keyWord` is empty just do nothing